composeApp/src/commonMain/kotlin/org/ooni/probe/domain/RunTestSuites.kt

```diff
@@ -0,0 +1,105 @@
+package org.ooni.probe.domain
+
+import co.touchlab.kermit.Logger
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.first
+import org.ooni.engine.Engine.MkException
+import org.ooni.engine.models.Result
+import org.ooni.engine.models.TaskOrigin
+import org.ooni.engine.models.TestType
+import org.ooni.probe.data.models.Descriptor
+import org.ooni.probe.data.models.NetTest
+import org.ooni.probe.data.models.ResultModel
+import org.ooni.probe.data.models.RunSpecification
+import org.ooni.probe.data.models.TestState
+import org.ooni.probe.data.models.UrlModel
+import kotlin.time.Duration.Companion.seconds
+
+class RunTestSuites(
```

Comment:
We are trying to move away from the `TestSuite` naming to `Descriptors` > `Nettests`